digdag-standards/src/main/java/io/digdag/standards/operator/pg/PgConnection.java

```diff
@@ -41,6 +41,7 @@ public void executeReadOnlyQuery(String sql, Consumer<JdbcResultSet> resultHandl
         try {
             execute("SET TRANSACTION READ ONLY");
             try (Statement stmt = connection.createStatement()) {
+                loggingExecuteSQL(sql);
                 ResultSet rs = stmt.executeQuery(sql);  // executeQuery throws exception if given query includes multiple statements
```

Comment:
Almost logging added in `AbstractJdbcConnection.java`. but this method use `stmt.executeQuery` at this place.
That's why I called `loggingExecuteSQL` method in this part.